flixel/system/FlxBasePreloader.hx

```diff
@@ -39,14 +44,31 @@ class FlxBasePreloader extends NMEPreloader
 	 * Defaults to 0.
 	 */
 	public var siteLockURLIndex:Int = 0;
-	
+
+	/**
+	 * The title text to display on the sitelock failure screen.
+	 * NOTE: This string should be reviewed for accuracy and may need to be localized.
```

Comment:
Added clarification on how to customize as well as a providing a simple example.